rpm/rpm_test.go

```diff
@@ -94,6 +94,16 @@ func TestRPM(t *testing.T) {
 	assert.NoError(t, err)
 }
 
+func TestWithRPMTags(t *testing.T) {
+	var info = exampleInfo()
+	info.RPM = nfpm.RPM{
+		Group:  "default",
+		Prefix: "/usr",
+	}
+	var err = Default.Package(info, ioutil.Discard)
```

Comment:
maybe write to a file and compare with the expected result? (like goldenfiles above)

Reply:
I added a new commit. Is that what you had in mind?

Reply:
looks great, thanks